### ISSUE TITLE
few comments

### ISSUE THREAD

**Issue by djaydev** [geekofweek/homeassistant#12]
I have the same exact iHome ISP6X Smart Plug and I agree they are very unreliable with the iHome app.  I bought them for probably the same reason you did, they said "works with Wink".  Hopefully you didn't throw them away because I recently discovered that they work great with homeassistant via [homekit](https://www.home-assistant.io/integrations/homekit/).  I don't even have them connected to the ihome app anymore and they haven't disconnected or anything since.  Let me know if you want any advice with getting them connected to HA homekit.

I've been searching all over the internet trying find someone who uses the Lutron Aurora Smart Bulb Dimmer with home-assistant and google led me to you.  Unfortunately I see now that you use the philips hue hub instead of "ZHA".  With your automation skills I was wondering if I could run something by you.
The Lutron Aurora Smart Bulb Dimmer sends events to home-assistant when the outer knob is turned like this:
```
{
    "event_type": "zha_event",
    "data": {
        "unique_id": "ff:ff",
        "device_ieee": "ff:ff:",
        "endpoint_id": 1,
        "cluster_id": 8,
        "command": "move_to_level_with_on_off",
        "args": [
            110,
            2
        ]
    },
```
Whether I turn it left or right, the only difference in the whole event is the "args" first number. That first number in the args increases or decreases in value based on turning left or right.  Everything else is the same, so I'm trying to figure out how to write an automation saying if the first number increases brighten light and vice versa.  Any thoughts? Sorry for the random out-the-blue.

**Comment by geekofweek** [comment]
Is the args value part of the device state?  If so it might be possible to break it off via a template sensor and then trigger off of that.

I've got a couple [examples like that](https://github.com/geekofweek/homeassistant/blob/master/sensors.yaml#L419), but not directly with the Aura Dimmer since I just stuck with the Hue Hub.  It's was one of my first devices and rock solid reliable so didn't see the need to replace it.

I donated the iHome's to a friend, I really like the Wemo's but the HomeKit integration is a solid idea.  That's how I integrated my Garage Door Opener, unfortunately I parted with them prior to that integration being released.  I have enough spare Wemo's laying around that it isn't much of a worry.
